cli.py

```diff
@@ -0,0 +1,107 @@
+"""
+Copyright (c) Facebook, Inc. and its affiliates.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+"""
+
+import functools
+import logging
+from typing import Callable, TypeVar
+
+import click
+from click_option_group import optgroup
+
+T = TypeVar("T")
+
+
+def _logging_options(func: Callable[..., T]) -> Callable[..., T]:
+    """
+    Common CLI options related to logging.
+    """
+
+    @optgroup.group("Logging")
```

Comment:
I'm planning to use groups for all related options. In this example:
```
Usage: cli.py download [OPTIONS]

Options:
  Logging: 
    --log-level [ERROR|WARNING|WARN|INFO|DEBUG]
                                  The minimum level of log statements to print.  [default: INFO]
  --help                          Show this message and exit.
```
you can see that `Logging` is a wrapper around the "--log-level" option (and any others that we add to the group in the future). Some of these commands are going to have enough options that groups should make the help text a little easier to read.